run-test262.c

```diff
@@ -2021,27 +2014,28 @@ char *get_opt_arg(const char *option, char *arg)
 
 int main(int argc, char **argv)
 {
-    int optind, start_index, stop_index;
+    int i, optind;
     BOOL is_dir_list;
     BOOL only_check_errors = FALSE;
     const char *filename;
     const char *ignore = "";
     BOOL is_test262_harness = FALSE;
     BOOL is_module = FALSE;
 
-#if !defined(_WIN32)
```

Comment:
I guess you removed this because we are not building the binary on windows yet, right?

Reply:
That's right. run-test262 didn't (and doesn't) build on win32 anyway so no point in the ifdef guard.

Reply:
Turns out it did build with mingw because mingw apparently includes pthreads?

I changed it to `#if !defined(__MINGW32__)`, I think that also covers mingw-w64.